manifests/e2etests.yaml

```diff
@@ -170,6 +170,30 @@ spec:
 ---
 apiVersion: v1
 kind: Namespace
+metadata:
```

Comment:
Don't need the namespace or the quota since this doesn't create resources or depend on namespaces being there